OperatingSystem-Services/Service-SOC/BasicDocumentation/DockerInstall-Wazuh.md

```diff
@@ -0,0 +1,48 @@
+# How to install Docker For Wazuh Purposes
+
+## DISCLAIMER: Do not follow docker docs, follow provided steps
+
+
+Follow these steps:
+
+## Increase max_map_count on your Docker host:
+
+`sudo sysctl -w vm.max_map_count=262144`
+
+### Or to do this permenantly do: 
+
+Edit `/etc/sysctl.conf` file, 
+
+Add `vm.max_map_count=262144` to the end of the file
+
+Run `sysctl vm.max_map_count` to make sure the number is 262144
+
+## Check Kernel Version
+
+Run: `sudo uname -r`
+
+Make sure kernel version is greater than 3.10
+
+
+## Run docker installation script:
+
+`sudo curl -sSL https://get.docker.com/ | sh`
+
+## Start Docker:
+
+`sudo systemctl start docker`
+
+# Install Docker Compose
```

Comment:
The single `#` header should probably only be used once in a Document as the title all others should be 2+.